doc/sphinx/source/utils/RTW/add_a_recipe.rst

```diff
@@ -26,48 +26,80 @@ How to add a recipe to the |RTW|
      [...]
      YYYY-MM-DD HH:MM:SS:sss UTC [12345] INFO    Run was successful
 
-#. Add the recipe to the ``[task parameters]`` section in the
-   ``esmvaltool/utils/recipe_test_workflow/flow.cylc`` file.
+.. _run_a_recipe_in_the_rtw:
 
-   .. hint::
-      If the recipe takes less than 10 minutes to run then it should be added
-      to the ``fast`` option. Recipes that take longer than ten minutes should
-      be added to the ``medium`` option.
+Run a Recipe in the |RTW|
+-------------------------
 
-   .. hint::
-      The line added should follow the format of ``recipe_new_recipe, \``,
-      unless the line is the last one in the list, in which case the line added
-      should follow the format of ``recipe_new_recipe``.
-
-#. If the duration of the recipe is larger than the value specified by the
-   ``execution time limit`` option in the ``[[COMPUTE]]`` section in the
-   aforementioned site-specific ``.cylc`` file, and / or the memory usage of
-   the recipe is larger than the value specified by the ``--mem`` option in the
-   ``[[[directives]]]`` section in the ``[[COMPUTE]]`` section, add a section
-   (in alphabetical order) to this file as shown below (round the duration to
-   the nearest second)::
-
-     [[process<fast=recipe_albedolandcover>]]
-     # Actual: 0m31s, 2.5 GB on 2024-04-08.
-     execution time limit = PT2M
-     [[[directives]]]
-         --mem = 3G
+#. The recipe should now be added to your ``<site>-recipes.cylc`` file. (Where
+   ``<site>`` is your site). Find this in the
+   ``esmvaltool/utils/recipe_test_workflow/site/`` directory. Add the recipe in
+   alphabetical order for easy user reference:
+   :ref:`Currently tested recipes <currently_tested_recipes>`.
+
+#. ``<site>-recipes.cylc`` contains **two lists of dictionaries**. The lists
+   are ``FAST_RECIPES`` and ``MEDIUM_RECIPES``.
```

Comment:
Each point must describe an instruction. This point contains only information, which means it would be better located in an `info` (or equivalent) box.

Reply:
@chrisbillowsMO and I worked together on the documentation and added the additional information to various e.g. `note` boxes: a353bf7 🎉